projects/public-search/src/app/api/item-api.service.ts

```diff
@@ -59,7 +54,7 @@ export class ItemApiService {
       ? `holding.pid:${holdings.metadata.pid} AND issue.status:${IssueItemStatus.RECEIVED}`
       : `holding.pid:${holdings.metadata.pid}`;
     return this._recordService
-      .getRecords('items', query, page, itemsPerPage, undefined, { view: viewcode }, this._headers, 'enumeration_chronology')
+      .getRecords('items', query, page, itemsPerPage, undefined, { view: viewcode }, BaseApi.reroJsonheaders, 'enumeration_chronology')
```

Comment:
small rebase conflict in prevision : https://github.com/rero/rero-ils-ui/pull/543